app/components/views/FatalErrorPage.js

```diff
@@ -20,13 +25,23 @@ class FatalErrorPage extends React.Component {
             }
           </div>
           <div className="fatal-error-title"><T id="fatal.suggestion.title" m="Suggested action to resolve error" />:</div>
-          <div className="fatal-error-area">
-            <div className="fatal-error-suggestion">
-              {daemonError.indexOf("resource temp") > 0 ?
-                <T id="fatal.suggestion.resources" m="This error typically means you have another instance of daemon running.  You should check your taskmanager or profiler to shutdown any still running daemon and then try again." /> :
+          <div className="fatal-error-suggestion">
+            {daemonError.indexOf(resourcesUnavailableError) > 0 ?
+              <T id="fatal.suggestion.resources" m="This error typically means you have another instance of daemon running.  You should check your taskmanager or profiler to shutdown any still running daemon and then try again." /> :
+              daemonError.indexOf(corruptedError) > 0  || daemonError.indexOf(checkSumError) ?
+                <Aux>
+                  <div className="fatal-error-reset-blockchain">
+                    <T id="fatal.suggestion.corrupted" m="This error means your blockchain data has somehow become corrupted.  Typically, this is caused by a sector on the HDD/SDD that went bad and its built-in SMART didn't repair it, or the more likely case, there was a memory issue which corrupted the data.  To resolve, you must delete your blockchain data and re-download.  Press the button below to complete the process. When you restart Decrediton, it will automatically begin your blockchain download. Please come to our support channel on slack/matrix/discord/rocketchat to get advice about running disk utilities. " />
+                  </div>
+                  <RemoveDaemonButton
+                    className="fatal-remove-button"
+                    modalTitle={<T id="fatal.removeConfirmModal.title" m="Remove daemon data"/>}
+                    modalContent={<T id="fatal.removeConfirmModal.content" m="Warning this action is permanent! Please make sure you want to remove your blockchain data before proceeding."/>}
```

Comment:
Ok, final note: Add a note to the effect of: "Decrediton will automatically shutdown after deleting the folder. Please manually restart it afterwards" to let users know that the app shutdown will happen.

Reply:
Good call, sec